Dockerfile-cypress

```diff
@@ -1,15 +1,23 @@
-FROM cypress/included:3.6.0
+FROM cypress/included:3.8.1
 
 ENV APP_HOME /usr/src/app
 RUN mkdir -p $APP_HOME
 WORKDIR $APP_HOME
 
-COPY . $APP_HOME
+COPY . ./
```

Comment:
If we are changing this then lines 3-5 can be removed

Reply:
reverted changes